### PR TITLE
[WIP] Basic artist profiles

### PR DESCRIPTION
**Pull request by JasoonS**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "@apollo/react-ssr": "^3.1.5",
     "@baransu/graphql_ppx_re": "^0.7.1",
     "@polkadot/util": "^2.8.1",
+    "@transak/transak-sdk": "^1.0.26",
```
Comment (JasoonS) [inline package.json]: Ah, annoying, the transak install is from the other branch...

File: src/Styles.re [hunk 2]
```diff
@@ -30,7 +29,7 @@ let app =
 let textOnlyModalText = style([padding(em(2.))]);
 let totalRaisedText = size => style([fontSize(em(size))]);
 
-let copyButton = style([float(`right), zIndex(50)]);
+let copyButton = style([Css.float(`right), zIndex(50)]);
```
Comment (JasoonS) [inline src/Styles.re:32]: I was getting a warning that I was overshadowing the name `float` which is also a type not just for css. This fixes that warning.

File: src/components/Dapp.re [hunk 3]
```diff
@@ -501,6 +502,18 @@ module DetailsViewAnimal = {
...
+             "Art by: "->React.string
+             <a
+               onClick={e => {
+                 ReactEvent.Mouse.preventDefault(e);
+                 clearAndPush("/#artist/" ++ artistInfo##id);
+               }}>
+               {artistInfo##name->React.string}
+             </a>
+           </p>
+         )}
```
Comment (JasoonS) [inline src/components/Dapp.re:516]: I show the artist's profile if an artist exists

File: src/components/OrgProfile.re [hunk 4]
```diff
@@ -179,18 +179,33 @@ module OrgPage = {
...
+            currentUsdEthPrice->Option.mapWithDefault(0., usdEthRate =>
+              mainnetEth->Eth.getFloat(Eth.Usd(usdEthRate, 2))
+            )
+            +. maticDai->Eth.getFloat(Eth.Eth(`ether))
+          )
+          ->Js.Float.toFixedWithPrecision(~digits=6),
+          mainnetEth->Web3Utils.fromWeiBNToEthPrecision(~digits=4)
+          ++ " ETH + "
+          ++ maticDai->Web3Utils.fromWeiBNToEthPrecision(~digits=2)
+          ++ " DAI",
```
Comment (JasoonS) [inline src/components/OrgProfile.re:198]: This was another (unrelated) fix, now it shows the org profiles Dai and Eth earning.

We can definitely make this better, but seems good for now.

File: src/components/StaticContent/Partners.re [hunk 5]
```diff
@@ -97,7 +97,7 @@ module OrgDetails = {
               ]),
             ])
           )
-          src={Animal.cdnBase ++ logo}
+          src={CONSTANTS.cdnBase ++ logo}
```
Comment (JasoonS) [inline src/components/StaticContent/Partners.re:100]: Lots of these changes, I just moved the cdn string to the `CONSTANTS` file, makes more sense there!

File: src/components/UserProfile.re [hunk 6]
```diff
@@ -18,7 +18,7 @@ module Token = {
     let clearAndPush = RootProvider.useClearNonUrlStateAndPushRoute();
     let image = Animal.useAvatar(tokenId);
 
-    <div className=Css.(style([width(vh(12.))]))>
+    <div className=Css.(style([width(vh(12.)), cursor(`pointer)]))>
```
Comment (JasoonS) [inline src/components/UserProfile.re:21]: Makes the animals on both the user and artist profiles display that they are clickable.

File: src/harberger-lib/Config.re [hunk 7]
```diff
@@ -0,0 +1,58 @@
+[@bs.val] [@bs.scope ("window", "location")]
```
Comment (JasoonS) [inline src/harberger-lib/Config.re]: Sorry, didn't mean for this change to be in this PR... Let me see if I can fix it

File: src/components/ArtistProfiles.re [hunk 8]
```diff
@@ -0,0 +1,306 @@
...
+  Css.(
+    style([
+      media(
+        "(max-width: 831px)",
+        [alignItems(center), justifyContent(center)],
+      ),
+    ])
+  );
+
+module ArtistDetails = {
```
Comment (JasoonS) [inline src/components/ArtistProfiles.re:13]: The layout and structure of this component is very similar to the UserProfiles

File: src/harberger-lib/QlHooks.re [hunk 9]
```diff
@@ -1174,3 +1246,81 @@ let useMetaTx = () => {
...
+            )
+          };
+
+        | None => ()
+        }
+      })
+    ->ignore;
+    dict->Js.Dict.values;
+  });
+};
```
Comment (JasoonS) [inline src/harberger-lib/QlHooks.re:1326]: All the new graphql hooks for artists

**Review by moose-code — APPROVED** [review]
Looks good!

**Pull request merged**